ci-operator/config/shiftstack/merge-bot/shiftstack-merge-bot-main.yaml

```diff
@@ -425,6 +425,34 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+- as: openstack-resource-controller-main
+  cron: 0 12 * * Mon,Thu
+  steps:
+    test:
+    - as: openstack-resource-controller-main
+      commands: |
+        merge-bot --source https://github.com/k-orc/openstack-resource-controller:main \
```

Comment:
This is a problem: we should not be merging downstream from main.

Reply:
* https://github.com/openshift/release/pull/61207
* https://github.com/openshift/openstack-resource-controller/pull/6